lib/api/bmm.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:bccm_core/bccm_core.dart';
+import 'package:brunstadtv_app/providers/settings.dart';
+import 'package:dio/dio.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:bmm_api/bmm_api.dart';
+
+final bmmApiProvider = Provider<BmmApiWrapper>((ref) {
+  return BmmApiWrapper(
+    basePathOverride: 'https://bmm-api.brunstad.org',
+    getAuthToken: () => ref.read(authStateProvider).auth0AccessToken,
+    interceptors: [
+      InterceptorsWrapper(onRequest: (options, handler) {
+        options.headers['Authorization'] = 'Bearer ${ref.read(authStateProvider).auth0AccessToken}';
+        options.headers['Accept-Language'] = [ref.read(settingsProvider).appLanguage.languageCode];
+        options.headers['BMM-Version'] = '1.101.0';
```

Comment:
Bør denne hard-kodes på denne måten?

Reply:
🤷

Reply:
Maybe good to change it to something else after agreement from karsten but that can be done after merge